Orelongz/WeConnect
# verify user account through mail

#### What does this PR do?
verify user account

#### Description of Task to be completed?
1. Send verification mail to a newly signed up user
2. Clear verification notification when user is verified

#### How should this be manually tested?
1. visit `https://weconnect-orelongz.herokuapp.com/`
2. Signup
3. Check mailbox used to register

#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
#158738409

#### Screenshots (if appropriate)
N/A
